processing/report_status.py

```diff
@@ -98,13 +100,35 @@ def enter_new_state(self, process_id, state_id):
         response = self._basic_post_request_core(msg, self.report_status_ws)
         return response.getcode() == HTTPStatus.OK
 
-    def register_siteRes_process(self, site_id, resolution):
-        req_data = {'SITE_ID': site_id,
+    def register_format_qaqc_process(
+            self, site_id, upload_id, process_timestamp,
+            prior_process_id=None, zip_process_id=None):
+        register_format_qaqc_ws = f'{self.register_qaqc_ws}/format_qaqc'
+        req_data = {'site_id': site_id,
+                    'upload_id': upload_id,
+                    'process_timestamp': process_timestamp,
+                    'code_version': self.code_version,
+                    'processor': getpass.getuser()}
+
+        if prior_process_id:
+            req_data.update({'prior_process_id': prior_process_id})

```

Comment:
You should be able to just do update(prior_process_id=prior_process_id)

Reply:
done